learning-pathways/dev_tools_training.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: learning-pathway
+title: Tool development for a nice & shiny subdomain
+type: admin-dev-community
+description: |
+  Discover Galaxy's communities and learn how to  create your subdomain and enrich it by writing, testing and submiting your tools on Galaxy. This learning pathway
+  will guide you through all the steps required to build a tool for Galaxy with Planemo for batch tools and how write an interactive tool.
+cover-image: assets/images/galaxy_subdomain.png
+cover-image-alt: Image of a researcher or developer on a computer thinking of building a community.
+editorial_board:
+- Marie59
+
+tags: [subdomain, community, tool development, 3-day course]
+
+pathway:
+
+  - section: "Day 1: Set up your subdomain for your community"
+    description: This first part explains how to discover the already existing communities (to avoid replication), how to build your subdomain, and finally how to set up your community
+    tutorials:
+      - name: sig_define
+        topic: community
+      - name: create_subdomain
```

Comment:
this tutorial doesn't exist, did you mean [this one](https://training.galaxyproject.org/training-material/topics/admin/tutorials/subdomain/tutorial.html)?

```suggestion
      - name: subdomain
```

Reply:
aha yes ... Thanks !